lib/msf/core/module/alert.rb

```diff
@@ -99,7 +99,6 @@ def add_alert(level, msg, &block)
     end
   end
 
-  # @nodoc
```

Comment:
Looks like this syntax is from rdoc; We should probably add docs here? 🤔 

Relevant thread: https://github.com/lsegal/yard/issues/11

Reply:
Probably would be best. There are a number of places that still need docs. Currently we are sitting at around 47% of our total library code being documented by YARD, so there is a lot of work to improve on. I'm mainly focused for now on making YARD run without errors or warnings when we build the documentation files, but this is definitely a good call out for something we should look to add in.

Another aspect would also be reviewing our existing documentation as I did see a few places where things were somewhat funky with our explanations and likely in need of a cleanup.